dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jGraph.java

```diff
@@ -197,4 +205,52 @@ public void execute(Command c) {
 	public void shutdown() {
 		service.shutdown();
 	}
+
+	@Override
+	public void addAnnotations(AnnotationParser source) {
+		execute(e -> addAnnotations(source));
+	}
+
+	@Override
+	public void addAnnotation(Annotation a) {
+		Map<String, Object> parameters = new HashMap<>(2);
+		parameters.put("from", a.getStart());
+		parameters.put("to", a.getEnd());
+		execute(e -> {
+			Node annotation = e.createNode(NodeLabels.ANNOTATION);
+			annotation.setProperty(PropertyTypes.ID.name(), a.getGeneName());
+			annotation.setProperty(PropertyTypes.STARTREF.name(), a.getStart());
+			annotation.setProperty(PropertyTypes.ENDREF.name(), a.getEnd());
+			annotation.setProperty(PropertyTypes.SENSE.name(), a.isSense());
+
+			ResourceIterator<Node> nodes = service.execute(GET_RANGE, parameters).columnAs("n");
+			nodes.forEachRemaining(n -> n.createRelationshipTo(annotation, RelTypes.ANNOTATED));
+		});
+		System.out.println("Finished loading annotations.");
```

Comment:
The Graph interface now extends the AnnotationCollection interface, therefore it has to implement all annotation methods.

A nice extra is that theoretically it is still possible to first load the graph and then the annotations. It's just terribly slow.